pip-dep/requirements.txt

```diff
@@ -12,7 +12,7 @@ psutil==5.7.0
 requests~=2.22.0
 RestrictedPython~=5.0
 scipy~=1.4.1
-syft-proto==0.4.7
+git+https://github.com/gmuraru/syft-proto.git@d8a4069457f65891ff02ad7e565dace8cd7b9a3f#egg=syft-proto
```

Comment:
This needs to be merged first.
```OnnxModel``` has protobuf + serde support.

Reply:
Merged to syft-proto you can update the requirement to master :)

Reply:
v0.4.10 of syft proto has been released with ONNX support

Reply:
Done -- waiting for tests